### DISCUSSION TITLE
How to use tests?

### DISCUSSION THREAD

**Discussion by olynch** [nrdxp/nixdev#479] [Q&A]
In the docs here: https://digga.divnix.com/tests.html, it gives a reference to a couple functions relating to testing. But I'm not sure where to use these functions, and where to put them in my flake.nix/other files, and I can't see this being used in the template. Is there a simple example of using tests that I could work off of? Thanks.

**Comment by teutat3s** [comment]
Maybe our recent first steps using tests could be helpful for you? https://git.b12f.io/pub-solar/os/pulls/91/files
